### ISSUE TITLE
General Object Info missing for datasets

### ISSUE THREAD

**Issue by bendichter** [HDFGroup/hdfview#321]
In HDFView 3.1.2 on macOS, when you examined a Dataset you would have two tabs, "Object Attribute Info" and "General Object Info". Now, in HDFView version 3.3.2, there are no tabs, and the display resembles the "Object Attribute Info" tab from the previous version. Is there a way to access the "General Object Info" information in the new version? It contains important information about the layout of the dataset.

<img width="1062" alt="image" src="https://github.com/user-attachments/assets/eef25631-2ca6-4ae4-bdc0-b621b736d51b">

<img width="1062" alt="image" src="https://github.com/user-attachments/assets/cbf6a4ff-9eb5-4bba-8864-ed41f33f01e5">

**Comment by byrnHDF** [comment]
Yes there should be two tabs, and there are two on windows and linux versions.
As I don't have a mac myself, this is news as no other report has been sent since the release. Therefore, we will need some help.

**Comment by bendichter** [comment]
@byrnHDF I'd be happy to jump on a call to show you how it looks on my end

**Comment by byrnHDF** [comment]
I have just tested this on a mac I have available - but it is not the latest, 12.7.6 on Intel processor. Everything appears fine.

**Comment by bendichter** [comment]
It may be an apple silicon issue then

**Comment by byrnHDF** [comment]
The next version will have a binary built on apple silicon using a Github runner.

**Comment by bendichter** [comment]
Cool, that sounds like it might fix it
